.github/workflows/sync-canary.yml

```
@@ -24,18 +25,61 @@ jobs:
-            git checkout -B canary
-            git pull origin main
-            git push origin canary --force
-          else
-            # Create canary branch if it doesn't exist
-            git checkout -b canary
-            git push origin canary
-          fi
+          # Create a unique branch for this sync
+          SYNC_BRANCH="sync-main-to-canary-$(date +%Y%m%d-%H%M%S)"
+          git checkout -b $SYNC_BRANCH
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)